src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -1026,41 +1011,40 @@ mod integration_tests {
             line
         };
 
-        // ------------------ past
+        // Too old event (> 14 days)
         add_event(Duration::days(-15));
-        // ------------------ 14 day limit
-        lines.push(add_event(Duration::days(-13)));
-        lines.push(add_event(Duration::days(-1)));
-        // ------------------ now
+        // Too new event (> 2 hours)
+        add_event(Duration::minutes(125));
+        // Now and future in +2h limit
         lines.push(add_event(Duration::zero()));
-        // ------------------ future
         lines.push(add_event(Duration::hours(1)));
         lines.push(add_event(Duration::minutes(110)));
-        // ------------------ 2h limit
-        add_event(Duration::minutes(125));
+        // 14 days limit
+        lines.push(add_event(Duration::days(-1)));
+        lines.push(add_event(Duration::days(-13)));
```

Comment:
I changed lines order here because:
- In function `process_events` events sorted by timestamp -- `events.sort_by_key(|e| e.timestamp);` to buckets 24h each
- In `CloudwatchFuture` (`request.rs`) sorted events in buckets pushed to aws from end

i.e. recent events pushed first, is this correct? Or oldest should be pushed first?